modules/monitoring-certificates.adoc

```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+//
+// * authentication/certificates/certificate-reference.adoc
+
+[id="monitoring-certificates_{context}"]
+= Monitoring certificates
```

Comment:
You know, when I first saw this heading, I thought it was a reference section for certificates related to monitoring. 

I'd suggest trying to make it stand out differently somehow. Maybe consider making it first/incorporating it into an intro area to the assembly or something?